Apps/WebClient/src/ClientApp/src/components/calendar/header.vue

```diff
@@ -105,6 +108,10 @@ export default class CalendarComponent extends Vue {
     private dispatchEvent() {
         let startDate = DateUtil.getMonthFirstDate(this.headerDate);
         this.$emit("update:currentDate", startDate);
+        this.eventBus.$emit(
```

Comment:
Can we use @Emit() for this similar to how we emit data from a child component in other parts of the app? See the communication modals for example.

Reply:
for info, this.eventBus.$emit needs to be used for sending communication between sibling (i.e. Calendar and Timeline list in this scenario). this.$emit is used for child vue to communicate with its parent.